pkg/web/src/pages/Settings/cluster/EditClusterModal.tsx

```diff
@@ -338,8 +338,12 @@ export const EditClusterModal = React.memo(() => {
                       value={cluster.discount}
                       onChange={(value: InputValue) => {
                         let discount = value;
+                        let reg = /^\d+(\.{0,1}\d+){0,1}$/
```

Comment:
Not a good way to implement it.

Reply:
Of course, this is definitely not a good way to implement it. There are two main reasons why I write regular expressions directly inside components.
1. The web side has a low utilization rate for its own encapsulation method, and it seems to have multiple files, which is rather messy, which is one of the reasons why I put forward the improvement of "READE.MD" in the issue
2. My implementation idea is based on the current code style. If I repackage this thing into a method for import verification, the amount of code generated is too much.